### PR TITLE
Fixing formatting in one sentence

### PR DESCRIPTION
**Pull request by Koziolek**
One small fix in EN version.

### CODE REVIEW

File: lessons/advanced/gen-stage.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ As we've read, the role we give our stage is important.  The GenStage specificat
 
 + `:consumer` — A sink.  A consumer requests and receives data from producers.
 
-Notice that our producers _wait_ for demand?  With GenStage our consumers send demand upstream and process the data from our producer.  This faciliates a mechanism known as back-pressure.  Back-pressure puts the onus on the producer to not over-pressure when consumers are busy.
+Notice that our producers __wait__ for demand?  With GenStage our consumers send demand upstream and process the data from our producer.  This facilitates a mechanism known as back-pressure.  Back-pressure puts the onus on the producer to not over-pressure when consumers are busy.
```
Comment (doomspork) [inline lessons/advanced/gen-stage.md]: @Koziolek do we use __bold__ or _italics_ in the other lessons?

Comment (Koziolek) [inline lessons/advanced/gen-stage.md]: @doomspork, __bold__ in most casses that's why I looks strange for me in this place.

Comment (doomspork) [inline lessons/advanced/gen-stage.md]: Thank you @Koziolek 👍

**Comment by doomspork** [comment]
Thank you @Koziolek, I appreciate your help!

**Pull request merged**